src/authProvider.component.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, RenderResult, screen, waitFor } from '@testing-library/react';
+import { AuthProvider, useAuthorisationState } from './authProvider.component';
+import React from 'react';
+
+const localStorageGetItemMock = vi.spyOn(
+  window.localStorage.__proto__,
+  'getItem'
+);
+
+const ConfigTest: React.FC = (): React.ReactElement => {
+  const authorisationState = useAuthorisationState();
+
+  // return authorisationState as a string to inspect later in tests
+  return (
+    <div data-testid="authorisationState">
+      Authorised: {JSON.stringify(authorisationState)}
+    </div>
+  );
+};
+
+describe('AuthProvider', () => {
+  // Create wrapper for authorisation tests
+  const renderComponent = (): RenderResult =>
+    render(
+      <AuthProvider>
+        <ConfigTest />
+      </AuthProvider>
+    );
+
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('returns correct authorisation state for admin user', async () => {
```

Comment:
Could these use the new ADMIN_ROLE_TOKEN now? I think I saw another place as well. I am guessing it cant be imported for cypress.